components/eam/src/chemistry/mozart/mo_usrrxt.F90

```diff
@@ -585,466 +348,34 @@ subroutine usrrxt( rxt, temp, tempi, tempe, invariants, h2ovmr,  ps, &
-         end if
-         if( usr_CO21_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO21_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO22_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO22_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO23_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO23_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO24_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO24_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO25_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO25_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO26_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO26_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO27_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO27_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO28_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO28_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO29_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO29_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO30_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO30_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO31_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO31_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO32_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO32_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO33_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO33_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO34_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO34_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO35_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO35_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO36_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO36_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO37_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO37_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO38_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO38_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO39_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO39_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO40_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO40_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO41_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO41_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-         if( usr_CO42_OH_ndx > 0 ) then
-            rxt(:ncol,:,usr_CO42_OH_ndx) = rxt(:ncol,:,usr_CO_OH_b_ndx)
-         end if
-      end if
-!lke--
 
-      deallocate( sfc_array, dm_array )
+    enddo level_loop
```

Comment:
Are you planning to refactor `usrrxt_hrates` later?

Reply:
@brigaudet and I talked on Slack and it seems like this routine is not used, so we can remove it.

Reply:
Done.